lib/docker-delta.coffee

```diff
@@ -120,11 +132,13 @@ exports.applyDelta = (srcImage, dstImage) ->
 		Promise.attempt ->
 			switch dockerDriver
 				when 'btrfs'
-					btrfs.deleteSubvolAsync(dstRoot)
-					.then ->
-						btrfs.snapshotSubvolAsync(srcRoot, dstRoot)
+					if srcRoot?
```

Comment:
Should the `if srcRoot?` be moved outside the `switch` rather than copying/pasting it in each branch?